generate_workflow.py

```diff
@@ -71,12 +96,43 @@ def sanitize_job_name(name):
     return "_" + h.hexdigest()
 
 
-def get_steps(build):
+def tuxsuite_setups(build_set, tuxsuite_yml):
```

Comment:
> Is the idea that we'll have separate tux suite configs for defconfigs vs all{yes|mod}configs?

Yes (well same configuration file, just separate sets of builds) and in the future, maybe even distribution configs. If we build the defconfigs and all*configs in the same build step, we have to wait for all builds to complete before we can start checking the results. By splitting up the configurations into separate build sets, we can have two `tuxsuite` invocations running at the same time, which would allow us to boot the defconfig images while the all*config build is still running.